Wordle.API/Wordle.API/Models/Player.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Wordle.Api.Models
+{
+    [Table("Player")]
```

Comment:
```suggestion
    [Table("Players")]
```
This should be plural